task_1.py

```diff
@@ -196,7 +191,7 @@ def main():
 
     
     
-    # TODO: Create loggers for wandb - ideally, use flags since wandb makes it harder to debug code.
+    # TODO (Q1.3): Create loggers for wandb - ideally, use flags since wandb makes it harder to debug code.
 
 
     for epoch in range(args.start_epoch, args.epochs):
```

Comment:
The next line, "adjust_learning_rate" is deprecated according to [this Piazza thread](https://piazza.com/class/kyiph2whvsp5i8/post/159_f12). We can remove it.

Reply:
Done